op-challenger/game/fault/trace/prestates/multi_test.go

```diff
@@ -161,6 +165,24 @@ func parseURL(t *testing.T, str string) *url.URL {
 	return parsed
 }
 
+func prestateHTTPServer(ext string) *httptest.Server {
+	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if strings.HasSuffix(r.URL.Path, ext) {
+			_, _ = w.Write([]byte(r.URL.Path))
+		} else {
+			w.WriteHeader(http.StatusNotFound)
+		}
+	}))
+}
+
+func makePrestateReader(path string) (io.ReadCloser, error) {
+	if strings.HasSuffix(path, ".gz") {
+		return os.Open(path)
+	} else {
+		return ioutil.OpenDecompressed(path)
+	}
```

Comment:
Isn't this equivalent to:
```suggestion
	return os.Open(path)
```
?

Reply:
fixed. thanks.